### PR TITLE
Add traditional Chinese translation to guide

### PR DESCRIPTION
**Pull request by wang-steven**
Add traditional Chinese version.

### CODE REVIEW

File: README_zh-TW.md [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+
+[![Say Thanks!](https://img.shields.io/badge/Say%20Thanks-!-1EAEDB.svg)](https://saythanks.io/to/RomuloOliveira)
+
+這是一份了解 commit 訊息的重要性和如何更好的撰寫它的指導手冊。
+
+它可以幫助你了解什麼是 commit、為什麼撰寫好的訊息很重要、最好的實踐案例以及一些技巧來計劃和 (重新) 撰寫良好的 commit 紀錄。
+
+## 什麼是“commit”？
+
+簡單來說，commit 就是在本地資料庫中撰寫的檔案的 _快照_。與印象中不同的是，[git 不僅儲存不同版本檔案之間的差異，還儲存了所有檔案的完整版本](https://git-scm.com/book/eo/v1/Ekkomenci-Git-Basics#Snapshots,-Not-Differences)。對於兩個 commit 之間沒有被修改的檔案，git 只儲存指向前一個完全相同的檔案的鏈結。
```
Comment (jiminhsieh) [inline README_zh-TW.md]: > 檔案的鏈結

How about `連結`？

File: README_zh-TW.md [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+![](https://i.stack.imgur.com/AQ5TG.png)
+
+## 為什麼 commit 資訊很重要？
+
+- 加快和簡化 code review
+- 幫助理解一個變更
+- 解釋那些無法只透過程式碼來描述的“為什麼”
+- 幫助未來的維護人員明白為什麼以及如何產生的變更，從而使問題排查和除錯更容易
+
+為了最大化這些結果，我們可以使用下一節中描述的一些好的實踐方式和標準。
```
Comment (jiminhsieh) [inline README_zh-TW.md]: > 為了最大化這些結果

How about `為了最好的效果`?

File: README_zh-TW.md [hunk 3]
```diff
@@ -0,0 +1,431 @@
...
+If applied, this commit will <commit message>
+如果通過，這個 commit 將會 <commit 訊息>
+```
+
+例子：
+
+```
+# Good
+If applied, this commit will use InventoryBackendPool to retrieve inventory backend
+如果通過，這個 commit 將使用 InventoryBackendPool 獲取庫存
```
Comment (jiminhsieh) [inline README_zh-TW.md]: > 獲取庫存

How about `取得存貨`?

File: README_zh-TW.md [hunk 4]
```diff
@@ -0,0 +1,431 @@
...
+```
+
+```
+# Bad
+If applied, this commit will used InventoryBackendPool to retrieve inventory backend
+如果通過，InventoryBackendPool 將會被用在獲取庫取
```
Comment (jiminhsieh) [inline README_zh-TW.md]: > 獲取庫取

ditto

File: README_zh-TW.md [hunk 5]
```diff
@@ -0,0 +1,431 @@
...
+
+### 範本
+
+下面是參考範本，最初由 [Tim Pope](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) 撰寫，出現在 [_Pro Git Book_](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project) 中。
+
+```
+用 50 個左右或更少的字來描述變更
+
+如有必要，可以提供更詳細的補充說明，並儘可能將其限定在每行 72 個字左右。
+在某些情況下，第一行被視為 commit 的標題，其餘路分被視為內容主體。
```
Comment (jiminhsieh) [inline README_zh-TW.md]: > 其餘路分

剩下(or 其餘)的文字

File: README_zh-TW.md [hunk 6]
```diff
@@ -0,0 +1,431 @@
...
+如若不然，在使用命令列，如 “log”，“shortlog” 以及 “rebase” 的時候，將會很容易混淆。
+
+解釋當前 commit 所解決的問題。
+請重點描述產生此變更的原因，而非方式（程式碼解釋了一切）。
+是否存在副作用以及其他隱含的影響？
+請在這裡將其解釋清楚。
+
+接下來請另起一行。
+
+ - 也可以使用枚舉要點的格式。
```
Comment (jiminhsieh) [inline README_zh-TW.md]: > 枚舉要點

How about we use the original English phrase - bullet point?
After googling, I found `枚舉` will be translated to `enum`.

File: README_zh-TW.md (hunk 1 above)
Comment (wang-steven) [reply README_zh-TW.md]: Fixed in [a1f5756](https://github.com/RomuloOliveira/commit-messages-guide/pull/48/commits/a1f575674b3202623565b80e1705522355b4298d)

File: README_zh-TW.md (hunk 2 above)
Comment (wang-steven) [reply README_zh-TW.md]: Fixed in [a1f5756](https://github.com/RomuloOliveira/commit-messages-guide/pull/48/commits/a1f575674b3202623565b80e1705522355b4298d)

File: README_zh-TW.md (hunk 3 above)
Comment (wang-steven) [reply README_zh-TW.md]: Fixed in [a1f5756](https://github.com/RomuloOliveira/commit-messages-guide/pull/48/commits/a1f575674b3202623565b80e1705522355b4298d)

File: README_zh-TW.md (hunk 4 above)
Comment (wang-steven) [reply README_zh-TW.md]: Fixed in [a1f5756](https://github.com/RomuloOliveira/commit-messages-guide/pull/48/commits/a1f575674b3202623565b80e1705522355b4298d)

File: README_zh-TW.md (hunk 5 above)
Comment (wang-steven) [reply README_zh-TW.md]: Fixed in [a1f5756](https://github.com/RomuloOliveira/commit-messages-guide/pull/48/commits/a1f575674b3202623565b80e1705522355b4298d)

File: README_zh-TW.md (hunk 6 above)
Comment (wang-steven) [reply README_zh-TW.md]: As you mentioned, I think '列舉' would be much better.
[a1f5756](https://github.com/RomuloOliveira/commit-messages-guide/pull/48/commits/a1f575674b3202623565b80e1705522355b4298d)

Comment (jiminhsieh) [reply README_zh-TW.md]: It sounds good to me. :)

**Review by RomuloOliveira — APPROVED** [review]
Thank you @wang-steven for your translation and @jiminhsieh for your review.

**Comment by RomuloOliveira** [comment]
Sorry, I always mess up forks when I try to rebase them :P your translation is merged, though: https://github.com/RomuloOliveira/commit-messages-guide/blob/master/README_zh-TW.md

**Pull request closed without merging**